app/templates/reminders.hbs

```diff
@@ -1,10 +1,17 @@
 {{outlet}}
-{{#each model as |reminder|}}
-  <ul>
-    <h2>
-      {{#link-to "reminders.reminder" reminder.id class='spec-reminder-item'}}
+{{#if model}}
+  {{#each model as |reminder|}}
+    <ul>
```

Comment:
You're iterating over each of your reminders and creating an entire `<ul>` just to display the h2...probably dont need the `<ul>` here at all.